### DISCUSSION TITLE
Remove deployment after new Production GH Action

### DISCUSSION THREAD

**Discussion by sebalaini** [vercel/vercel#10522] [Help] [answered]
### Question

I know I can use the Vercel CLI to delete all deployments that don't have an active preview URL or production domain with `vercel remove my-project --safe`

Is there a way to automate this process via GH actions every time a new production has been created?

**Comment by sebalaini** [comment]
bump, is there any GH action for this?

**Comment by leerob** [comment, marked as answer]
You can now configure the retention period for deployments through the dashboard and CLI.

For example, canceled and errored deployments might be set to 30 days retention, while production deployments might be set to 1 year. Recently deleted deployments are shown in your project settings and can be instantly restored within 30 days of deletion.

Learn more in our [documentation](https://vercel.com/docs/security/deployment-retention).
